### ISSUE TITLE
systemd/dbus broken again

### ISSUE THREAD

**Issue by peterfraedrich** [coreos/go-systemd#311]
It looks like (similar to #295 ) dbus is broken again.

```# github.com/coreos/go-systemd/dbus
../../../../pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190620071333-e64a0ec8b42a/dbus/dbus.go:127:33: cannot use dbus.SystemBusPrivate (type func() (*dbus.Conn, error)) as type func(...<T>) (*dbus.Conn, error) in argument to dbusAuthHelloConnection
../../../../pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190620071333-e64a0ec8b42a/dbus/dbus.go:136:33: cannot use dbus.SessionBusPrivate (type func() (*dbus.Conn, error)) as type func(...<T>) (*dbus.Conn, error) in argument to dbusAuthHelloConnection
../../../../pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190620071333-e64a0ec8b42a/dbus/dbus.go:146:42: undefined: dbus.ConnOption
../../../../pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190620071333-e64a0ec8b42a/dbus/dbus.go:204:48: undefined: dbus.ConnOption
../../../../pkg/mod/github.com/coreos/go-systemd@v0.0.0-20190620071333-e64a0ec8b42a/dbus/dbus.go:224:53: undefined: dbus.ConnOption
```
My env:
* Ubuntu 18.04
* go version go1.12.4 linux/amd64

```$> go env
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/pete/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/pete/go"
GOPROXY=""
GORACE=""
GOROOT="/usr/lib/go"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/pete/go/src/xxxxx.com/xxxxx/xxxxx/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build564163905=/tmp/go-build -gno-record-gcc-switches"
```

**Comment by peterfraedrich** [comment]
additionally, because this repo doesn't use semantic versioning for releases, i am unable to downgrade to the previous release using `go get ...`

**Comment by lucab** [comment]
Thanks for the report. The error you are seeing is due to a non-matching (i.e. ancient) version of go-dbus in your dependency tree. You can fix it by updating it to anything >= 5.0.

If you are using `dep`, this library provides a Gopkg manifest to make that automatic.

If you are using go module, you need to adjust that manually on your own. https://github.com/coreos/go-systemd/pull/307 has the whole backstory on that.
